Gems/Atom/Feature/Common/Assets/Shaders/ScreenSpace/DeferredFog.azsl

```diff
@@ -12,15 +12,25 @@
 #include <Atom/Features/ScreenSpace/ScreenSpaceUtil.azsli>
 
 #include <scenesrg.srgi>
+#include <viewsrg.srgi>
+
+#ifndef ENABLE_FOG_LAYER
+#define ENABLE_FOG_LAYER    1
+#endif
```

Comment:
I wonder if it would be useful to separate fog layer support from fog noise support.

Reply:
Not in our use case at this moment.